### DISCUSSION TITLE
cannot create an access token because i donot find « Developer settings »l

### DISCUSSION THREAD

**Discussion by jeanrichet** [community/community#75551] [New to GitHub]
Hello,
I cannot push my local because of : 
 :jano@jeanrichet-X751NV:~/Bureau/mesitesweb/lurcy1$ git push -u origin main
Username for 'https://github.com': jeanrichet
Password for 'https://jeanrichet@github.com': 
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Échec d'authentification pour 'https://github.com/jeanrichet/lurcy1.git/'

So I try to generate a personal acces token. I fail to do it because my GUI does not give me access to  « Developer settings » in the list of the links on the left side of my screen. I attach the screenshot of my interface where i look for « Developer settings ».
![githubessailurcyscreenshot.png](https://github.zendesk.com/attachments/token/58yC0DWbguHCIU0qvKhMxnk8h/?name=githubessailurcyscreenshot.png)
Thanks to anyone who can help me creating my token by finding and using  « Developer settings » in the list of the links on the left side of my screen 
Jean Richet
jean.richet@orange.fr

**Comment by AidenJamesYt** [comment]
follow these steps:

Navigate to your account settings:

Click on your profile picture in the top right corner of any GitHub page.
Select "Settings" from the dropdown menu.
Access developer settings through URL:

In the URL bar of your browser, append "/settings/tokens" to the end of the "Settings" page URL.
For example, if your settings page URL is https://github.com/settings/profile, then the modified URL would be https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens.
Generate a personal access token:

Click on the "New personal access token" button.
Give your token a descriptive name.
Select the appropriate scopes for the token based on the tasks you want to perform.
Click on the "Generate token" button.
Copy and save your personal access token:

The generated token will be displayed on the screen.
Copy and save the token securely in a password manager or text file.
Keep the token confidential as it provides access to your GitHub account.
Once you have generated a personal access token, you can use it to authenticate with GitHub instead of your password. For example, to push your local repository to a remote repository, you would use the following command:

git push -u origin main https://<personal_access_token>@github.com/<username>/<repository_name>.git
Replace <personal_access_token> with your actual personal access token, <username> with your GitHub username, and <repository_name> with the name of your repository.

**Reply by johnnyxtheman** [reply]
so there are 2 settings option ???? thas fxx confuzing and searcyh for h and h to find thid fxxx developer .. be more pesific where the fxxx thins are !!!

**Comment by elderguardian** [comment]
Hey, just click this link: https://github.com/settings/tokens
